src/ngraph_mark_for_clustering.cc

```diff
@@ -439,7 +492,7 @@ Status MarkForClustering(Graph* graph) {
         break;
       }
 
-#if (TF_VERSION_GEQ_1_11)
+#if (TF_ENABLE_DEADNESS_CHECK)
```

Comment:
I would recommend renaming this to NGRAPH_TF_DISABLE_DEADNESS_CHECK and reverse the logic i.e., `#if !defined(NGRAPH_TF_DISABLE_DEADNESS_CHECK)`

Reply:
Ok. Would this be an environment variable NGRAPH_TF_DISABLE_DEADNESS_CHECK?

Reply:
That's what I am leaning towards. By using this logic - if we can turn off the deadness check (for some experiments/tests) - then that's good. If not then I guess what you have is fine